### ISSUE TITLE
Suggestion: Don't place scenery below supports in scatter placement mode

### ISSUE THREAD

**Issue by ipatix** [OpenRCT2/OpenRCT2#3098]
**OS: Linux x86_64 - Debian Sid** 
**Version: 0.0.4.0 build fc2d48d** 
**Commit: fc2d48d** 

So I really enjoy putting time into making the supports of my wodden roller coasters look good. However, due to visibilty reasons I tend to remove all trees before that process. So after finishing my the supports (scenery) I'd like to just place all the trees and plants with the scatter tool. This unfortunately places trees below all the supports (of the rollercoaster and scenery) and that way I have to remove them all again which can get pretty annoying.
So, would it be possible to implement something that prevents trees/scenery being placed by scatter mode under anything that has supports below?

- [ ] Reproducable in RCT2 (vanilla)?
- [ ] Multiplayer?

**Steps to reproduce:**
1. Build a roller coaster and/or place some additional support structures beside
2. Place tree's with the scatter tool and see the mess

**Screenshots / Video:**
![scr9](https://cloud.githubusercontent.com/assets/8502545/13550687/b2a7c830-e323-11e5-8e8d-c4b33fc41ce8.png)

**Save game:**
N/A

**Comment by duncanspumpkin** [comment]
I don't see there being enough reason to implement this.

**Comment by ipatix** [comment]
Honestly, it's nothing too bad and I think I can deal with this kind of thing. It was just a convenient thing I had on my mind.

**Comment by RapleAQN** [comment]
I agree with this! Make it toggleable in the settings if its gonna be implemented

**Comment by Wirlie** [comment]
It's a good idea! This makes the game more realistic (In the real life, a support is not built over a tree)... however, it's true what @RapleAQN says, you can implement a setting for those who prefer retain the previous functionality with the supports.

**Comment by PFCKrutonium** [comment]
IMO if this is changed it shouldn't have a setting - At least not one in the GUI.
